### PR TITLE
rename storageclassclaim to storageclaim in gRPC methods

### PR DESCRIPTION
**Pull request by leelavg**
Depends on red-hat-storage/ocs-operator#2548

### CODE REVIEW

**Comment by leelavg** [comment]
/hold

for dependencies & custom go.mod replace.

File: go.mod
Comment (nb-ohad) [inline go.mod:5]: This need to go away before merging

File: controllers/storageclaim_controller.go
Comment (nb-ohad) [inline controllers/storageclaim_controller.go:291]:
```suggestion
			storageClaimStorageType = providerclient.StorageTypeSharedFile
```

**Review by nb-ohad — CHANGES_REQUESTED** [review]

File: go.mod
Comment (leelavg) [reply go.mod]: done.

File: controllers/storageclaim_controller.go
Comment (leelavg) [reply controllers/storageclaim_controller.go:280]: will be taken up in a different PR to cut across all the components and change to `SharedFile` and `Block` as the values.

**Pull request merged**
